modules/_labs/color-picker/react/lib/parts/Swatch.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {colors, borderRadius} from '@workday/canvas-kit-react-core';
+import {pickForegroundColor} from '@workday/canvas-kit-react-common';
+
+import {checkSmallIcon} from '@workday/canvas-system-icons-web';
+import {SystemIcon} from '@workday/canvas-kit-react-icon';
+
+const whiteColor = colors.frenchVanilla100;
+
+export interface ContainerProps extends React.HTMLAttributes<HTMLDivElement> {
+  color: string;
+  style?: React.CSSProperties;
+}
+
+export const Container = styled('div')<ContainerProps>(
```

Comment:
The type of element has come up before. Should it be a `button`, `li`, or `div`. Least semantic is `div`. We could easily argue it is a part of a list if the field (target element) states this is some type of combobox with a menu. This way as long as the target has focus, keyboard works (up, down, possibly left/right).

We may need a deeper talk with accessibility to find out what we should do here. I think `div` is fine for now until we get greater direction.

Reply:
Oh. Wait. I didn't catch that these are treated like actual buttons. We had to add `tabIndex="0"` to get focus to work. Perhaps the decision isn't necessarily on this component. I see `SwatchContainer` elsewhere that also has a `div` with an `onClick` and a `tabIndex="0"` to make it act like a focusable button. We'll have to sync up with accessibility to know if all these swatches should have individual focus or the focus management and keyboard management comes from the target of this popup (and that target has a `aria-popup="true"`.

More info on the `combobox` role: https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.0pattern/combobox-autocomplete-list.html

It is up to Accessibility which role and therefore behavior the color picker field target element should get.